doc/api/api_changes/2015-08-16_remove_lena_from_sample_data.rst

```diff
@@ -0,0 +1,6 @@
+Removed `Lena` images from sample_data
+``````````````````````````````````````
+
+The lena.png and lena.jpg images have been removed from matplotlibs sample_data 
+directory. The images are also no longer available from 
+`matplotlib.cbook.get_sample_data`
```

Comment:
Can you add 'We suggest usinge `matplotlib.cbook.get_sample_data('grace_hopper.png')` or `matplotlib.cbook.get_sample_data('grace_hopper.jpg')` instead.' ?